src/app/@ansyn/ansyn/components/ansyn-footer/ansyn-footer.component.spec.ts

```diff
@@ -19,10 +19,11 @@ describe('AnsynFooterComponent', () => {
 		selector: 'ansyn-overlay-navigation-bar'
 	});
 
+	beforeEach(waitForAsync(() => {
```

Comment:
Is this addition needed?

Reply:
in angular11, the previous "async" method is redundent that is for not to confuse with the general "async" for js

Reply:
I am asking about something else. Why did you add `beforeEach`? It does not seem required.

Reply:
I guess it happened automatically, its fixed now thanks :)